sqlserver/tests/common.py

```diff
@@ -46,38 +50,7 @@ def get_local_driver():
     'password': 'Password123',
     'tags': ['optional:tag1'],
     'include_task_scheduler_metrics': True,
-}
-
-INSTANCE_AO_DOCKER_PRIMARY = {
```

Comment:
Deleted this extra config info to make it easier to add new config options to `INSTANCE_DOCKER` without needing to add to every existing instance.